packages/react/src/primitives/Icon/icons/IconClose.tsx

```diff
@@ -9,13 +9,13 @@ import { InternalIcon } from './types';
  * @internal For internal Amplify UI use only. May be removed in a future release.
  */
 export const IconClose: InternalIcon = (props) => {
-  const { className, size, ...rest } = props;
```

Comment:
The `size` prop is used by other components besides the `FieldClearButton`. Is there a different approach to be taken here?

Reply:
I think we can leave it here, but the change above I believe will still work to fix the issue.